roles/cockpit/tasks/main.yml

```diff
@@ -1,13 +1,16 @@
 ---
 - name: Install cockpit-ws
+  when: not openshift.common.is_containerized | bool
   package: name={{ item }} state=present
   with_items:
     - cockpit-ws
     - cockpit-system
     - cockpit-bridge
     - cockpit-docker
     - "{{ cockpit_plugins }}"
-  when: not openshift.common.is_containerized | bool
+  register: l_result
+  retries: 4
+  until: l_result.rc | default(-1) == 0 or l_result.results is defined and 'No package matching' in l_result.results.0
```

Comment:
This one has logic to avoid retries when the package really just isn't there. However this relies on the output from the backing impl and that seems kind of brittle to me. Plus this is a lot to paste everywhere.